src/startupscriptgenerator/src/dotnetcore/main.go

```diff
@@ -130,6 +130,9 @@ func main() {
 		}
 
 		common.WriteScript(fullOutputPath, command)
+
+		userRunCommand := common.ParseUserRunCommand(fullAppPath + "/appsvc.yaml")
```

Comment:
For all of the places where you're doing this (creating a full file path), I would recommend using [`filepath.Join()`](https://www.geeksforgeeks.org/filepath-join-function-in-golang-with-examples/) as these types of libraries typically shield you from the OS being used and whether you should be using `/` or `\` in the path.